.github/workflows/build.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.11", "3.12"]
+        python-version: ["3.11", "3.12", "3.13"]
```

Comment:
Don't forget to also add 3.13 to the pyproject.toml file list of supported versions (since this did build fine)

Reply:
Done